libnmstate/nm/bridge.py

```diff
@@ -107,7 +115,13 @@ def get_info(nmdev):
 
     port_profiles = _get_slave_profiles(nmdev)
     info[LB.CONFIG_SUBTREE] = {
-        LB.PORT_SUBTREE: _get_bridge_ports_info(port_profiles),
+        LB.PORT_SUBTREE: _get_bridge_ports_info(
+            port_profiles,
+            show_vlan_info=(
+                is_vlan_filtering_supported
```

Comment:
This should be `is_vlan_filtering_supported()`, right?

Reply:
> This should be `is_vlan_filtering_supported()`, right?

yes it should. 

Don't spend time on this, I'm (very) slowly working on this PR, and will be updating it in the following days. Much will have changed.